x-pack/solutions/observability/plugins/synthetics/docs/openapi/synthetic_apis.yaml

```diff
@@ -0,0 +1,1446 @@
+                        "key": "param1",
+                        "value": "value1"
+                      },
+                      {
+                        "id": "param2-id",
+                        "key": "param2",
+                        "value": "value2"
+                      }
+                    ]
+  /api/synthetics/params/{id}:
+    delete:
+      summary: Delete a parameter
+      operationId: delete-parameter
+      description: >
+        Delete a parameter from the Synthetics app.
+        You must have `all` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
+      tags:
+        - synthetics
+      parameters:
+        - in: path
+          name: id
+          description: The ID for the parameter to delete.
+          required: true
+          schema:
+            type: string
+    get:
+      summary: Get a parameter
+      operationId: get-parameter
+      description: You must have `read` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
+      tags:
+        - synthetics
+      parameters:
+        - in: path
+          name: id
+          description: The unique identifier for the parameter.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: A successful response.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/getParameterResponse"
+              examples:
+                getParameterResponseExample1:
+                  summary: Read access
+                  description: A successful response for a user with read-only permissions to get a single parameter.
+                  value: |-
+                    {
+                      "id": "unique-parameter-id",
+                      "key": "your-api-key",
+                      "description": "Param to use in browser monitor",
+                      "tags": ["authentication", "security"],
+                      "namespaces": ["namespace1", "namespace2"]
+                    }
+                getParameterResponseExample2:
+                  summary: Write access
+                  description: A successful response for a user with write permissions to get a single parameter.
+                  value: |-
+                    {
+                      "id": "unique-parameter-id",
+                      "key": "your-param-key",
+                      "description": "Param to use in browser monitor",
+                      "tags": ["authentication", "security"],
+                      "namespaces": ["namespace1", "namespace2"],
+                      "value": "your-param-value"
+                    }
+    put:
+      summary: Update a parameter
+      operationId: put-parameter
+      description: You must have `all` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
```

Comment:
```suggestion
      description: 
      Update a parameter in the Synthetics app.
      
      You must have `all` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
```

Reply:
Ditto fixed in fa994557ff351594bf7cbdccf473bea646391109